cmd/sealer/cmd/root.go

```diff
@@ -81,6 +89,13 @@ func initConfig() {
 	if rootOpt.debugModeOn {
 		logrus.SetLevel(logrus.DebugLevel)
 	}
+
+	if rootOpt.hideLogColor {
```

Comment:
```
logrus.SetFormatter(&logrus.TextFormatter{
			DisableColors: rootOpt.hideLogColor,
})
```
Would this be better?
I'm ok with both.

Reply:
u are right